src/_includes/config/setup-cron_2.3_how-to.md

```diff
@@ -44,3 +43,17 @@ content='The `update/cron.php` file exists in [Composer](https://glossary.magent
 
 In [Composer-based installations](https://glossary.magento.com/composer), Magento creates the `update/` directory when you run `composer create-project`. Running `composer install` does not create the `update/` directory (if it did not exist before). See [Recreate the Magento updater](https://devdocs.magento.com/guides/v2.3/comp-mgr/cli/cli-upgrade.html#recreate-magento-updater).'
 %}
+
+Starting from Magento 2.3.4 mentioned crons have been grouped to `default` (`cron:run`, `update/cron.php`, `setup:cron:run` crons) and `non-optional` (`cron:run` cron only) groups.
```

Comment:
We may play around with the word "default" here. The "default" CRON **[group](https://github.com/magento/magento2/blob/a31f4a35c018ad09654e5bd5871086b73fbd3d2d/app/code/Magento/ProductAlert/etc/crontab.xml#L9)** is an old good definition. So maybe we should not call it a **group** to avoid confusion. Also, it looks like "--non-optional -d(default)" also means [default](https://github.com/magento/magento2/blob/64d4b3781ae5d77bb237148378b00d261da4a647/app/code/Magento/Cron/Console/Command/CronInstallCommand.php#L60). 

> Install only the non-optional (default) tasks

But in general, the info is very useful, thank you!

Reply:
> We may play around with the word "default" here.

I've re-phrased the explanation. Could you please review a new option :)

Thanks, @rogyar !

Reply:
Much better. Thank you